### PR TITLE
fix(dashboardeditor): pass through pii props to the child components

### PR DESCRIPTION
**Pull request by scottdickerson**
Closes #

**Summary**

- All of these fixes are just to pass all the i18n props from the DashboardEditor layer down through to the underlying card components

**Change List (commits, features, bugs, etc)**

- just passing and adding any missing i18n props down

**Acceptance Test (how to verify the PR)**

- To verify it, look at the new i18n story for DashboardEditor and add and manage each of the card types to see the translated strings

### CODE REVIEW

File: src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx [hunk 1]
```diff
@@ -154,6 +156,30 @@ const DataSeriesFormItemModal = ({
   const { id, type } = cardConfig;
   const baseClassName = `${iotPrefix}--card-edit-form`;
 
+  const handleTranslation = useCallback(
+    (idToTranslate) => {
```
Comment (davidicus) [inline src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx:160]: can we align the use of these arguments. In CardEditFormContent.jsx we are using `id` and here we are using `idToTranslate`. I prefer the more explicit option.

**Review by davidicus — APPROVED** [review]
There is just the one comment about alignment of argument names but otherwise looks good.

File: src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx (hunk 1 above)
Comment (scottdickerson) [reply src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx:160]: sure

**Review by sstone2423 — APPROVED** [review]

**Pull request merged**